mobile/src/bundles/common/styles/global-styles.ts

```diff
@@ -0,0 +1,89 @@
+import { type FlexAlignType, type FlexStyle, StyleSheet } from 'react-native';
+
+import { AlignmentValues } from '~/bundles/common/enums/enums';
+
+const globalStyles = StyleSheet.create({
```

Comment:
To divide some logic, you can create some margins object, paddings object, flex object etc. and just spread it in globalStyles 
```
const globalStyles = StyleSheet.create({
	...flex,
	...margins,
	...paddings,
});
```

Reply:
Also, you can apply created global styles in app.tsx and remove styles.ts from there.

Reply:
I forgot about app.tsx and styles.ts for it ... I will do it after your comments

Reply:
Done